docker_run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+###  SET HOST FILE PATHS HERE. All paths must be absolute!  ###
+HOST_CFG=$(pwd)/rabbitmq-worker.conf
+HOST_LOG=$(pwd)/rabbitmq-worker.log
+HOST_ERR=$(pwd)/rabbitmq-worker.err
```

Comment:
Sorry, I don't understand why we would be better off redirecting to stdout & stderr. There would still be the need to redirect stdout/err to log files on the host.  All it would do is change where those files are specified.